.github/workflows/build.yml

```diff
@@ -148,24 +160,39 @@ jobs:
           
       - name: Run unit tests
         run: npm run test:unit
+        env:
+          CI: true
         working-directory: ./src/vscode-bicep
-        
+
       - name: Run E2E tests
+        run: npm run test:e2e
+        env:
+          CI: true
+        if: runner.os != 'Linux'
+        working-directory: ./src/vscode-bicep
+        
+      - name: Run E2E tests (Linux)
         run: xvfb-run -a npm run test:e2e
+        env:
+          CI: true
```

Comment:
>CI [](start = 10, length = 2)

Env vars can be set at the job level btw. I do that in the bicep job above.

Reply:
Moved it to job level.